services/migadu.lua

```diff
@@ -0,0 +1,28 @@
+local M       = {}
+
+M.name        = "Migadu"
+M.description = "Migadu is an email hosting provider."
+M.category    = "email"
+M.fields      = {}
```

Comment:
All parameters except `self` must be added in the `M.fields`. This will be used filled by user in the Web UI.